index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://www.google.com">My Logo</a>
+      <button>
+        <img src="img/Union.png" alt="wite bars">
+      </button> 
```

Comment:
- [ ] please kindly ensure The hamburger button is on the right, and has a `nav` tag.

* The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.
* kindly remove the button tag from the hamburger icon
![Screenshot (1021)_LI](https://user-images.githubusercontent.com/65298048/235293868-7778c6e5-48b2-4394-a7b5-058aebaf3948.jpg)